cmd/kube-proxy/app/server_others.go

```diff
@@ -243,7 +244,7 @@ func (s *ProxyServer) createProxier(config *proxyconfigapi.KubeProxyConfiguratio
 				s.Hostname,
 				s.NodeIPs,
 				s.Recorder,
-				s.HealthzServer,
+				[2]*healthcheck.ProxierHealthManager{s.HealthzServer.NewProxierHealthManager(), s.HealthzServer.NewProxierHealthManager()},
```

Comment:
Maybe it would be simpler to just keep passing `s.HealthzServer` to the proxiers, and let them call `NewProxierHealthManager()` (or rather, `NewProxierHealthUpdater()`) themselves?